README.md

```diff
@@ -1,3 +1,40 @@
 # DGtalTools-contrib
-Various tools from the community built using DGtal
+
+Various tools from the community built using DGtal. Contrary the to DGtalTools project, these tools are not necessary final but can be considered as development tools or prototypes used to share recent research in progress.
+
+
+This project is actually organized as follows:
+
+ - [Geometry2d](#geometry2d)
+
+ - [Geometry3d](#geometry3d)
+
+ - [Visualisation](#visualisation)
+
+
+
+Geometry2d
+----------
+
+This section can contains various utilities related to 2d geometry (2d estimators, 2d contour tools, ...):
+
+   - meaningFullThickness: to display the meaningful thickness of digital contour.
+
+
+Geometry3d
+----------
+
+As the previous section but in 3d, it contains actually these tools:
+
+   - basicEditMesh: to apply basic mesh edition (scale change, mesh face contraction, face filtering).
+   - compareErrorMesh: compute hausdorf distance between two meshes.
```

Comment:
does it really compute mesh hausdorff distance or it is hausdorff distance of point sets (vertices) ?